src/Stash/Interfaces/ItemInterface.php

```diff
@@ -134,4 +138,30 @@ public function getCreation();
      * @return \DateTime
      */
     public function getExpiration();
+
+
+    /**
+    * Sets the expiration based off of an integer or DateInterval
+    *
+    * @param int|\DateInterval $time
+    * @return static The invoked object.
+    */
+    public function expiresAfter($time);
+
+
+    /**
+    * Sets the expiration to a specific time.
+    *
+    * @param \DateTimeInterface $expiration
+    * @return static The invoked object.
+    */
+    public function expiresAt($expiration);
+
+
+    /**
+    * Persists the Item's value to the backend storage.
+    *
+    * @return bool
+    */
+    public function save();
```

Comment:
True, but there always has been one on the Item object of stash. I'm thinking of deprecating it but leaving it for now.